pyfolio/tests/test_perf_attrib.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
+import pandas as pd
+import unittest
+
+from pyfolio.perf_attrib import perf_attrib
+
+
+def generate_toy_risk_model_output():
+    """
+    Generate toy risk model output.
+
+    Returns
+    -------
+    tuple of (returns, factor_returns, positions, factor_loadings)
+    returns : pd.DataFrame
+    factor_returns : pd.DataFrame
+    """
+    start_date = '2017-01-01'
+    periods = 10
+    dts = pd.date_range(start_date, periods=periods)
+    np.random.seed(123)
```

Comment:
In that case I wonder if some sort of encoding scheme might be preferable to random numbers, but I don't think we need to worry about that at this point, since it seems like we're not using this yet?